Classes/WPEditorView.m

```diff
@@ -275,8 +275,17 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
             [newValue getValue:&newSize];
```

Comment:
>because of the slightly different name 

doH! It was a difference in the Mac and iOS docs and I didn't notice I was looking at that mac version. Sorry about that.